Services/Common/OJS.Services.Common/Implementations/PublisherService.cs

```diff
@@ -13,4 +14,8 @@ public PublisherService(IBus bus)
     public Task Publish<T>(T obj)
         where T : class
         => this.bus.Publish(obj);
+
+    public Task PublishMultiple<T>(IEnumerable<T> objs)
```

Comment:
Rename to `PublishBatch` as your method is just a wrapper to the `PublishBatch` from MassTransit